pyproject.toml

```diff
@@ -17,18 +17,19 @@ classifiers=[
     ]
 
 [tool.poetry.dependencies]
-python = ">=3.8,<3.12"
+python = ">=3.9, <4"
 requests =  "^2.31.0"
 requests-oauthlib = "^1.3.1"
 tenacity = "^8.2.2"
 tqdm = "^4.66.0"
 geojson = "^3.0.1"
 geopandas = "^0.13.2"
 pystac-client = "^0.7.2"
+pyproj = "^3.6.1"
```

Comment:
needed to add this for my local environment.
This is anyway a sub-dependecy of `geopandas`.

Happy to remove it if there are any concerns.